modules/ROOT/pages/project-best-practicesIndex.adoc

```diff
@@ -0,0 +1,26 @@
+= Project best practices
```

Comment:
I would prefer you to use camel case for the file name instead of kebab case, it's better to stick with the convention unless you are able to justify this choice :)

FYI: https://wprock.fr/blog/conventions-nommage-programmation/

Reply:
oops. Thx